README.md

```diff
@@ -31,7 +31,7 @@ go get github.com/intelligentpos/structextract
 		Field4: 123,
```

Comment:
Will fix the slice thing, good catch. I am trying to fix the snippet but it acts a little weird...